src/executables/util_cs.c

```diff
@@ -3258,15 +3264,15 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   do
     {
+      memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
+      strcpy (local_database_name, database_name);
+      strcat (local_database_name, "@localhost");
 
       if (check_applied_info)
 	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
 	  db_clear_host_connected ();
 
+	  printf ("\n *** Applied Info. *** \n");
 	  if (check_database_name (local_database_name))
```

Comment:
error check is required for printing error message at `check_applied_info_end`
`error = check_database_name (local_database_name)`

Reply:
`check_database_name_local()` prints error message. So we don't need to store the error message.